ios/RNShare.m

```diff
@@ -266,7 +266,7 @@ - (NSDictionary *)constantsToExport
         if (activityError) {
             [controller  dismissViewControllerAnimated:true completion:nil];
             failureCallback(activityError);
-        } else if (completed || activityType == nil) {
```

Comment:
The condition here is unnecessary as the completion is checked [https://github.com/krystofcelba/react-native-share/blob/80fc9967efb4162e19950eb0f209bc2e14a49231/index.js#L268](here) in JS code and in situations like canceled print it causes that the JS callback isn't called at all.